static/app/views/issueDetails/groupEvents.tsx

```diff
@@ -100,25 +99,9 @@ function UpdatedSearchBar({
     }, {});
   }, [data]);
 
-  const tagValueLoader = useCallback(
-    (key: string, search: string) => {
-      const orgSlug = organization.slug;
-      const projectIds = [group.project.id];
-
-      return fetchTagValues({
-        api,
-        orgSlug,
-        tagKey: key,
-        search,
-        projectIds,
-      });
-    },
-    [api, group.project.id, organization.slug]
-  );
-
   const getTagValues = useMemo(
-    () => makeGetIssueTagValues(tagValueLoader),
-    [tagValueLoader]
+    () => makeGetIssueTagKeyValues({organization, pageFilters}),
+    [organization, pageFilters]
```

Comment:
Updated issue details with the same logic since it also uses the `makeGetIssueTagValues` function. I have assumed with this change that issue details also pulls tags from `events` and `issuePlatform`, lmk if that is incorrect